include/phasar/Controller/AnalysisController.h

```diff
@@ -131,6 +131,10 @@ class AnalysisController {
 
   AnalysisController(AnalysisController &&) = delete;
 
+  AnalysisController &operator=(const AnalysisController &) = delete;
+
+  AnalysisController &operator=(const AnalysisController &&) = delete;
```

Comment:
```suggestion
  AnalysisController &operator=(const AnalysisController &) = delete;
  AnalysisController &operator=(const AnalysisController &&) = delete;
```
In the other classes we always put the 5 special member functions into a block without empty lines between them.
PS: needs to fixed for the above 3, github just does not allow me to include them in a suggestion :cry: